### DISCUSSION TITLE
Why is zendesk sell archived?

### DISCUSSION THREAD

**Discussion by throttleup** [airbytehq/airbyte#40701] [Connector Questions] [answered]
Can this source be resurrected? It looks like it has recent activity. If not, how can I install an archived connector

**Comment by marcosmarxm** [comment, marked as answer]
You can read more about Archived connectors [here](https://docs.airbyte.com/integrations/connector-support-levels#archived
).
It is possible to remove the archive status of the connector, but someone needs to update the connector to meet the latest requirements: low-code, poetry dependency, removal of the docker image. At the moment we don't have a specific documentation explaining how to do the process as we're still developing the CDK and connectos.
